STEERING-COMMITTEE.md

```diff
@@ -100,19 +97,25 @@ first name):
 \* Open seats have been allocated to the organization per the committee rules,
   but have not been assigned to a member of the organization.
 
-### Allocation of seats
+### Composition
+
+As of 2020, the three organizations with the largest demonstrated 
+commitment to the project are Google, IBM/RedHat, and VMWare. For
+2021, a 5 person steering committee should be competent to
+execute its responsibilities, with three seats for Google, one seat
```

Comment:
can you say a little more about the thinking behind the new committee going from 4:3 google/others to 3:2 google/others? Would it not seem a little unfortunate if a discussion about making steering more representative of the community ended up with it becoming _more_ dominated by one vendor?

Reply:
+1 -- Paul's proposal adds two seats for end-users to inform the community (via Steering) about how Knative is used in the world; this proposal would further reduce Steering's ability to hear feedback from the community (which feels like it's already a problem -- witness the 50+ people interested in the steering membership process with no current outlet).

Frankly, vendor seats have also been fairly disruptive to the community -- a huge amount of context has been lost in the last year as Google has shifted 100% of the seats assigned to it between different individuals, seemingly without a lot of handoff. This would be less of a problem if this wasn't *also* a majority bloc, but it feels like any progress that's been made over the last year on governance issues has been completely erased.

Reply:
(Including the actual process of conducting a TOC election, which was removed from the current charter!)

Reply:
Additionally, having more representation would align better with missing #1: “empowers contributors”. As presented here lots of contributors would likely feel disenfranchised IMO.